lib/python/treadmill_aws/sproc/garbage_collector.py

```diff
@@ -0,0 +1,53 @@
+"""Garbage collector for LDAP and IPA.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import time
+import click
+from treadmill import cli
+from treadmill import plugin_manager
+
+_LOGGER = logging.getLogger(__name__)
+_MODULE = 'treadmill_aws.garbage_collector'
+
+
+def _run_sync(gc_plugins, once, interval):
+    """Garbage collector plugins executor.
+    """
+    while True:
+        for name in gc_plugins or plugin_manager.names(_MODULE):
+            try:
+                plugin = plugin_manager.load(_MODULE, name)
+                plugin()
+            except Exception:  # pylint: disable=W0703
+                _LOGGER.exception('Error processing plugin: %s', name)
+
+        if once:
+            return
+        time.sleep(interval * 60 * 60)
+
+
+def init():
+    """Return top level command handler.
+    """
+    @click.command()
+    @click.option('--interval', default=1.0, type=float, help='in hours')
```

Comment:
make it seconds. all our intervals are in seconds, for consistency.

Reply:
done